spec/jobs/problem_checks_spec.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Jobs::ProblemChecks do
+  before do
+    AdminDashboardData.reset_problem_checks
+  end
+
+  after do
+    Discourse.redis.flushdb
+  end
+
+  it "runs the problem check that has been added and adds the messages to the load_found_scheduled_check_problems array" do
+    AdminDashboardData.add_scheduled_problem_check(:test_identifier) do
+      AdminDashboardData::Problem.maybe_create("big problem")
+    end
+
+    execute_job_class_with_args
```

Comment:
What do you think about just leaving this as `described_class.new.execute`, I find that it makes the spec easier to read because it makes it clear that we're testing the `#execute` instance method here.

Reply:
Yeah I guess, I was just annoyed at how often we were calling `described_class.new.execute(args)`, will just change back to that cause it is clearer